packages/web/src/components/data-entry/InputV2.tsx

```diff
@@ -59,20 +66,44 @@ export const InputV2 = (props: InputV2Props) => {
     [styles.error]: error
   }
 
+  /**
+   * Since Firefox doesn't support the :has() pseudo selector,
+   * manually track the focused state and use classes for required and disabled
+   */
```

Comment:
Oof.
* Change looks safe at surface level, but wondering if there are any other places using this `InputV2` component that should be tested?
* We should probably chat with client and follow up on any other code using `:has` selectors